apps/web/src/pages/get-started/tabs/Setup.tsx

```diff
@@ -0,0 +1,367 @@
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        const { environment } = useEnvironment();
+
+        return (
+          <TextElement>
+            Add <Code>NOVU_SECRET_KEY</Code> environment variable to your <Code>.env</Code> file
+            <CodeSnippet command={`NOVU_SECRET_KEY=${(environment as any)?.apiKeys[0].key}`} />
+          </TextElement>
+        );
+      },
+    },
+    {
+      title: 'Create a workflow',
+      content: () => (
+        <>
+          <TextElement>
+            Create a new workflows file at a novu folder at <Code>{workflowPath}</Code> that will contain your workflow
+            definitions. You can also organize your workflows in separate files
+          </TextElement>
+          <br /> <br />
+          <CodeEditor
+            height="400px"
+            readonly
+            setCode={() => {}}
+            code={`import { workflow } from '@novu/framework';
+import { z } from 'zod';
+
+export const testWorkflow = workflow('test-workflow', async ({ step, payload }) => {
+  await step.email('send-email', async (controls) => {
+      return {
+          subject: controls.subject,
+          body: 'This is your first Novu Email ' + payload.userName,
+      };
+  },
+  {
+      controlSchema: z.object({
+          subject: z.string().default('A Successful Test on Novu from {{userName}}'),
+      }),
+  });
+  }, {
+      payloadSchema: z.object({
+          userName: z.string().default('John Doe'),
+      }),
+  });
+});`}
+          />
+          <HStack gap="50" className={css({ color: 'typography.text.secondary', mt: '12px' })}>
+            <IconOutlineMenuBook />
+            <a href="https://docs.novu.co/workflow/introduction" target={'_blank'}>
+              Learn more on building workflows
+            </a>
+          </HStack>
+        </>
+      ),
+    },
+    {
+      title: 'Run the application server',
+      content: () => <CodeSnippet command={`cd my-novu-app && npm run dev`} />,
+    },
+    {
+      title: 'Run Novu Studio',
+      content: () => (
+        <TextElement>
+          Novu Studio is used to preview your local workflows, inspect controls and trigger test workflows.
+          <br />
+          To Start Novu Studio, run the following command in your terminal:
+          <br />
+          <br />
+          <CodeSnippet command={`npx novu@latest dev --port 4000`} />
+          <HStack gap="50" className={css({ color: 'typography.text.secondary', mt: '12px' })}>
+            <IconOutlineMenuBook />
+            <a href="https://docs.novu.co/workflow/studio" target={'_blank'}>
+              Learn more about Novu Studio
+            </a>
+          </HStack>
+        </TextElement>
+      ),
+    },
+    {
+      title: 'Send a test notification',
+      content: () => (
+        <TextElement>
+          Once the Studio is open, navigate to the 'test-workflow' tab and click on the 'Send test' button.
+        </TextElement>
+      ),
+    },
+  ];
+}
```

Comment:
I wonder if we need this step here. I know we want to encourage users to trigger, but the trigger CTA is also located in the Studio flow. I think the comment in step 5 ('Run Novu Studio'), about the trigger might be enough.

I'm concerned that having six steps is too many. Based on my suggestions, we could reduce it to four, although I'm not sure if that's the best solution for user onboarding.